website/src/pages/SinglePR/SinglePR.jsx

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+import { useParams } from "react-router-dom";
+import useApiCall from "../../utils/Hook";
+import RingLoader from "react-spinners/RingLoader";
+
+import "../SinglePR/singlePR.css";
+
+import { formatDate, errorApi } from "../../utils/Utils";
+
+const SinglePR = () => {
+  const { pull_nb } = useParams();
+
+  const {
+    data: dataSinglePr,
+    isLoading: singlePrLoading,
+    error: singlePrError,
+  } = useApiCall(`${import.meta.env.VITE_API_URL}pr/info/${pull_nb}`, []);
+
+  const isComparisonAvailable =
+    dataSinglePr.Base !== "" && dataSinglePr.Head !== "";
+
+  if (
+    dataSinglePr.error == "GET https://api.github.com/repos/vitessio/vitess/pulls/13675: 404 Not Found []"
+  ) {
+    return (
+      <div className="errorPullRequest">Pull request {pull_nb} not found</div>
+    );
+  }
+
+  return (
+    <div className="singlePR">
+      <div className="singlePR__container flex--column">
+        {singlePrError ? (
+          <div className="apiError">{errorApi}</div>
+        ) : singlePrLoading ? (
+          <div className="loadingSpinner">
+            <RingLoader loading={singlePrLoading} color="#E77002" size={300} />
+          </div>
+        ) : (
+          <>
+            <div className="singlePR__top flex">
+              <div>
+                <h2>
+                  [#{pull_nb}]{dataSinglePr.Title}
```

Comment:
```suggestion
                  [#{pull_nb}] {dataSinglePr.Title}
```

Reply:
let's add a space between the PR number and its title and can you add a link to the GitHub PR on `#{pull_nb}`